workflow-ui/compose/src/androidTest/java/com/squareup/workflow1/ui/compose/NoTransitionBackStackContainer.kt

```diff
@@ -0,0 +1,39 @@
+package com.squareup.workflow1.ui.compose
+
+import android.content.Context
+import android.view.View
+import android.view.ViewGroup.LayoutParams.MATCH_PARENT
+import com.squareup.workflow1.ui.BuilderViewFactory
+import com.squareup.workflow1.ui.ViewFactory
+import com.squareup.workflow1.ui.WorkflowUiExperimentalApi
+import com.squareup.workflow1.ui.backstack.BackStackContainer
+import com.squareup.workflow1.ui.backstack.BackStackScreen
+import com.squareup.workflow1.ui.backstack.R
+import com.squareup.workflow1.ui.bindShowRendering
+
+/**
+ * A subclass of [BackStackContainer] that disables transitions to make it simpler to test the
+ * actual backstack logic. Views are just swapped instantly.
+ */
+// TODO ship this?
+@OptIn(WorkflowUiExperimentalApi::class)
+internal class NoTransitionBackStackContainer(context: Context) : BackStackContainer(context) {
```

Comment:
@rjrjr I really think this should be the default behavior, and maybe we ship a subclass that uses the transition manager. Or maybe we don't. Either way, I am not convinced the default base class behavior should be to animate.

Reply:
Completely agree.

I also think that any configuration of transition animation should be decoupled from backstacks, it's more fundamental than that. In non-compose, I want to tie it to `WorkflowViewStub` instead. I might have a notion of how to use `ViewRegistry` for that, if I can ever land the work to generalize that.

Reply:
That sounds nice! Ok, I'll leave this copy in here for now, and will follow up to clean these up (#306).